Content.Server/GameTicking/Rules/Components/ZombieRuleComponent.cs

```diff
@@ -25,4 +25,21 @@ public sealed partial class ZombieRuleComponent : Component
     /// </summary>
     [DataField]
     public float ZombieShuttleCallPercentage = 0.7f;
+
+    /// <summary>
+    /// Is the automatic shuttle called.
+    /// </summary>
+    public bool ZombieShuttleCalled = false;
+
+    /// <summary>
+    /// The reason why the emergency shuttle is called.
+    /// </summary>
+    [DataField]
+    public string ShuttleCallReason = "zombie-shuttle-call";
```

Comment:
You should be using `LocId` for the type as it'll validate the localization string exists.

```suggestion
    public LocId ShuttleCallReason = "zombie-shuttle-call";
```